sample-code/javascript-webdriverio/package.json

```diff
@@ -4,14 +4,14 @@
   "description": "Sample tests using webdriverio",
   "main": "index.js",
   "scripts": {
-    "test": "mocha --timeout 6000000 test/basic"
+    "test": "./node_modules/bin/mocha --timeout 6000000 test/basic"
```

Comment:
This is not needed (or wanted). Commands run through the `"scripts"` section of the `package.json` have `./node_modules/.bin` added to the `PATH`. See https://docs.npmjs.com/cli/run-script